examples/Demo/FluentUI.Demo.Client/Layout/DemoNavMenu.razor.css

```diff
@@ -22,4 +21,6 @@ li {
 a {
   text-decoration: none;
   color: var(--colorNeutralForeground2);
+  display: block;
+  padding: 8px 10px;
```

Comment:
I prefer to keep the padding in the `li` tag to keep a space between the item and the browser border.

Reply:
Adding `display: block;` is enough: the user can click on the link and to the right of that link.
I don't agree with you trying to change the design without warning, without a screenshot.

Reply:
Especially on mobile you will notice the difference

Reply:
I agree with the problem but not the solution. 
Although the `NavMenu` component has not yet been developed in v5, it will replace this entire area once it is done.